### PR TITLE
Update changelog with notes for v1.4.2

### PR DESCRIPTION
**Pull request by MarcosDY**
Update changelog to v1.4.2

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## [1.4.1] - 2022-09-07
+
+### Added
+- Support to node reattestation to certain Node Attestors (#3031)
+- Server now adds unique identifier to SVIDs subject (#3367)
```
Comment (azdagron) [inline CHANGELOG.md:7]:
```suggestion
- The X509-SVID Subject field now contains a unique ID to satisfy RFC 5280 requirements (#3367)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+- Agents are shutting down when banned (#3308)
```
Comment (azdagron) [inline CHANGELOG.md:8]:
```suggestion
- Agents now shut down when banned (#3308)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+
+### Fixed
+- Vault Upstream Authority to prevent adding duplicate certificates (#3376)
```
Comment (azdagron) [inline CHANGELOG.md:11]: I've made a suggestion to a change. I'm not sure we need to include this in the CHANGELOG though. There really isn't any user impact. This was more of a cleanliness change from what I understand. The X509CAChain is not stored in the bundle. The only impact from what I understand is the contents of the journal... which users won't notice....

What do you think?

```suggestion
- Vault Upstream Authority no longer adds duplicate certificates to the CA chain using Vault 1.11+ (#3376)
```

**Review by azdagron — COMMENTED** [review]
Thanks for putting this together @MarcosDY !

File: CHANGELOG.md (hunk 3 above)
Comment (MarcosDY) [reply CHANGELOG.md:11]: mm I was not sure about adding it either...
I'm ok with removing it from changelog

**Review by evan2645 — APPROVED** [review]
LGTM

**Review by azdagron — APPROVED** [review]

**Pull request merged**